stob/module.c

```diff
@@ -23,26 +23,26 @@
 #include "stob/module.h"
 #include "module/instance.h"
 
-static int level_stob_enter(struct m0_module *module);
-static void level_stob_leave(struct m0_module *module);
+//static int level_stob_enter(struct m0_module *module);
+//static void level_stob_leave(struct m0_module *module);
 
-static const struct m0_modlev levels_stob[] = {
+/* static const struct m0_modlev levels_stob[] = {  
```

Comment:
Unused code can be marked as "UNUSED" to fix compiling error?

Reply:
Can you suggest how to adorn the unused code as UNSUSED with out commenting ?

Reply:
Please refer to:
lib/misc.h:#define M0_UNUSED __attribute__((unused))